test/screenshot/infra/lib/diff-base-parser.js

```diff
@@ -83,7 +83,7 @@ class DiffBaseParser {
       if (process.env.TRAVIS_BRANCH) {
         baseBranch = `origin/${process.env.TRAVIS_BRANCH}`;
       } else {
-        baseBranch = await this.gitHubApi_.getPullRequestBaseBranch(prNumber);
+        baseBranch = await this.gitHubApi_.getPullRequestBaseBranch(prNumber) || baseBranch;
```

Comment:
Wouldn't this cause incorrect diffs against base in cases where we're PR'ing against feat/typescript?

Also, this line suggests that getPullRequestBaseBranch just returns falsy in the problem case (it doesn't actually throw)?

Reply:
Yes, it would cause incorrect base diffs for PRs merging into non-master branches if the GitHub API returns an error (e.g., due to rate limiting on an unauthenticated account).

I removed all the `throw new Error()` statements from the `GitHubApi` class because GitHub integration isn't totally critical to screenshot tests, so we probably shouldn't cause Travis CI to fail when a status check `POST` fails.